src/network.cpp

```diff
@@ -38,6 +41,35 @@ inline String uptimeString() {
   return ret;
 }
 
+DynamicJsonDocument parseOwieStatusData() {
```

Comment:
Please name this function "generateOwieStatusJson()" or something like that as it doesn't do any parsing.